airconsole-1.9.0-alpha.js

```diff
@@ -1092,11 +1131,63 @@ AirConsole.prototype.init_ = function(opts) {
     version: me.version,
     device_motion: opts.device_motion,
     synchronize_time: opts.synchronize_time,
+    silence_players: me.silence_players,
     location: me.getLocationUrl_(),
     translation: opts.translation
   });
+  this.enablePlayerSilencing(me.silence_players);
 };
 
+/**
+ * Enables the player silencing
+ * @param {boolean} silence_players If true, the player silencing feature is active
+ * @private
+ */
+AirConsole.prototype.enablePlayerSilencing = function (silence_players) {
+  if (this.getCustomDeviceState(AirConsole.SCREEN, AirConsole.PLAYER_SILENCE_KEY) !== undefined)
```

Comment:
Custom device state can not have platform state. Custom device state is only for games (secret platform keys dont make sense).
Please add it to the device state (the parent dict). You can not simply access that once, you will need a helper function that calls the parent frame to do so (see how "players" does it)
make sure that you also clear that property once you switch game in the parent frame.

Reply:
@laurentndream Based on our shared plan, this would in this case likely also apply to your [SetImmersiveState](https://github.com/AirConsole/airconsole-api/blob/2f97da388a0b8904eaf38d325cfafb90cfe939ea/airconsole-1.9.0-alpha.js#L300) that takes the same approach.